tests/migrations/test_commands.py

```diff
@@ -1708,7 +1708,7 @@ class Meta:
                 call_command('makemigrations', 'migrations', interactive=True, stdout=out)
             output = out.getvalue()
             prompt_output = prompt_stdout.getvalue()
-            self.assertIn("You can accept the default 'timezone.now' by pressing 'Enter'", prompt_output)
+            self.assertIn("Accept the default 'timezone.now' by pressing 'Enter'", prompt_output)
```

Comment:
Can we add tests for other messages?

Reply:
All the functions that contain messages either call sys.exit or return ask_default function or return NOT_PROVIDED. Which messages exactly did you want to test?

Reply:
As many as we can :smile:

Reply:
Thanks. I think that's it. 😀